day_6/exercises/good_dog.rb

```diff
@@ -0,0 +1,23 @@
+class GoodDog
+  attr_accessor :name, :height, :weight
+
+  def intialize(n, h, w)
+    @name = n
```

Comment:
Always avoid using 1 word var or parameter names. It's best to be clear and air on the side of verbosity.